fortunate/fortunate.php

```diff
@@ -1,38 +0,0 @@
-<?php
```

Comment:
Why is this addon removed? You don't mention it in the summary.

I think you should mark it as deprecated instead if there is a valid reason to do so. Maybe a maintainer comes out of the shadows. If none is found, it can be removed with the release after the 2018.08 one were it was deprecated as with the other deprecated addons.

Reply:
This addon will stop working as soon as the `.htconfig.php` file will be (re)moved.

Reply:
True. And it is therefore deprecated. If somebody wants to fix it they can pick it up and if not it will be moved to the deprecated-addons repository a release later..

Reply:
Ugh.